components/Nav/Navbar.jsx

```diff
@@ -0,0 +1,35 @@
+import styled from 'styled-components';
+import Burger from './Burger';
+import Link from 'next/link'
+
+const Nav = styled.nav`
+  top: 0;   
+  width: 100%;
+  height: 5em;  
+  font-family: var(--title);
+  font-size: 2.5em;
+  background-color: black;
+  color: white;
+  padding: 0 20px;
+  padding-botton: 20px;
+  display: flex;
+  justify-content: space-between;
+  z-index: 10;
+  .logo {
+    padding: 15px 0;
+    cursor: pointer;
+  }
+`
+
+const Navbar = () => {
+  return (
+    <Nav>
+      <div className="logo">
```

Comment:
- Give a bit more spacing to the top and left of the logo

Reply:
ok